src/index.js

```diff
@@ -1,32 +1,62 @@
+//@flow
+
 import assert from 'assert';
+
+import FusionApp from 'fusion-core';
+import type {FusionPlugin} from 'fusion-core';
+
 import {mockContext, renderContext} from './mock-context.js';
 import simulate from './simulate';
 
-export function request(app, url, options = {}) {
+declare var __BROWSER__: boolean;
+
+const request = (app: FusionApp) => (
+  url: string,
+  options: * = {}
+): Promise<*> => {
   if (__BROWSER__) {
     throw new Error(
       '[fusion-test-utils] Request api not support from the browser. Please use `render` instead'
     );
   }
   const ctx = mockContext(url, options);
   return simulate(app, ctx);
-}
+};
 
-export function render(app, url, options = {}) {
+const render = (app: FusionApp) => (
+  url: string,
+  options: * = {}
+): Promise<*> => {
   const ctx = renderContext(url, options);
   return simulate(app, ctx);
+};
+
+export function getSimulator(app: FusionApp, testPlugin?: FusionPlugin<*, *>) {
+  if (testPlugin) {
+    app.register(testPlugin);
```

Comment:
Should we allow for an array of test plugins?

Reply:
This is just the public API from the DI system. Whether multiple registration makes sense is something that we can discuss on Giancarlo's new proposal.